app/assets/stylesheets/application.scss

```diff
@@ -17,5 +17,22 @@
  */
 .cancel {
   text-align: center;
+}
+
+body {
+ background-image: image_url("");
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores